README.md

```diff
@@ -94,28 +93,28 @@ follows:
     gmk@Centos-7.img demo> cat hello
     world
 
-Once the developer has completed their environment the image file can be
-compressed and copied to any other system that has Singularity installed.
+Once, the developer has completed their environment the image file can
```

Comment:
The original 'Once the developer' without comma is correct here.

Reply:
Yes @dctrud is correct.